packages/fcl/src/current-user/index.js

```diff
@@ -94,9 +96,11 @@ function notExpired(user) {
 }
 
 async function getAccountProofData() {
-  const accountProofDataResolver = await config.get("fcl.accountProof.resolver")
-  const accountProofData =
-    accountProofDataResolver ?? (await accountProofDataResolver())
```

Comment:
I don't even get how this worked before.  The way that microbundle was compiling this basically turned into
`accountProofData = accountProofDataResolver ? await accountProofDataResolver() : accountProofDataResolver` which is backwards and makes no sense.

Very bizarre.